src/Files.App/Views/Properties/GeneralPage.xaml

```diff
@@ -478,20 +502,24 @@
 					</Button.Flyout>
 				</Button>
 
-				<MenuFlyoutSeparator
+				<!--  (Divider)  -->
+				<Border
 					Grid.Row="3"
 					Grid.Column="0"
 					Grid.ColumnSpan="4"
-					Margin="-12,0"
-					Background="{ThemeResource CardStrokeColorDefaultBrush}" />
+					Height="1"
+					Margin="-16,0"
+					Background="{ThemeResource DividerStrokeColorDefault}" />
 
-				<!--  Disk cleanup  -->
+				<!--  Disk Cleanup  -->
 				<TextBlock
 					Grid.Row="4"
 					Grid.Column="0"
 					Grid.ColumnSpan="2"
+					Margin="4,0,0,0"
```

Comment:
Can you share a screenshot of the before and after?

Reply:
Wrong position 'Cleanup your drive contents' TextBlock

Before/After
![image](https://user-images.githubusercontent.com/62196528/232266164-3aaf1ea5-0345-4eb8-ae7c-5497c40c3bad.png)

Reply:
Good catch